lib/valkyrie/specs/shared_specs/persister.rb

```diff
@@ -24,6 +24,17 @@ class CustomResource < Valkyrie::Model
     expect(persister.save(model: resource).id).not_to be_blank
   end
 
+  it "stores created_at/updated_at" do
+    book = persister.save(model: resource_class.new)
+    book.title = "test"
+    book = persister.save(model: book)
+
+    expect(book.created_at).not_to be_blank
+    expect(book.updated_at).not_to be_blank
+    expect(book.created_at).not_to be_kind_of Array
+    expect(book.updated_at).not_to be_kind_of Array
```

Comment:
It would be nice to verify that these values are dates, and preferably recent dates if that's easy to do.